src/test/groovy/de/tracetronic/jenkins/plugins/ecutestexecution/util/ProcessUtilTest.groovy

```diff
@@ -0,0 +1,34 @@
+package de.tracetronic.jenkins.plugins.ecutestexecution.util
+
+import spock.lang.Specification
+
+class ProcessUtilTest extends Specification {
+    def 'test killProcess'(int timeout, expected) {
+        given:
+            boolean actual = ProcessUtil.killProcess("doesReallyNotExistFoo", timeout)
+
+        expect:
+            actual == expected
+
+        where:
+            timeout | expected
+                -1  |   true
+                0   |   true
+                1   |   true
+    }
```

Comment:
in complex tests use:
`given -> when -> then`

But here is it totally enough to use:
```
expect:
    ProcessUtil.killProcess("doesReallyNotExistFoo", timeout) == expected

where:
    timeout | expected
        -1  |   true
        0   |   true
        1   |   true
```